### PR TITLE
fix #65

### LINKED ISSUE #65: coloc.abf returns NaN for all posteriors
Hello,

Thanks for the great software! I was trying to use it on my data and it seems that I encounterd a situation where all posteriors are returned NaN but there is no reason / error reported for this. 

I checked the "check_dataset()" outputs for both the inputs, both report "## NULL" and I do get SNP priors & hypothesis priors but no posteriors. 

What I'm trying to run:

`> coloc.abf(dataset1=qtlD, dataset2=GwasCC)
PP.H0.abf PP.H1.abf PP.H2.abf PP.H3.abf PP.H4.abf 
      NaN       NaN       NaN       NaN       NaN 
[1] "PP abf for shared variant: NaN%"
Coloc analysis of trait 1, trait 2

SNP Priors
   p1    p2   p12 
1e-04 1e-04 1e-05 

Hypothesis Priors
        H0     H1     H2        H3      H4
 0.7534289 0.1115 0.1115 0.0124211 0.01115

Posterior
nsnps    H0    H1    H2    H3    H4 
 1115   NaN   NaN   NaN   NaN   NaN `

I'm not sure what is going on here and I hoped you might have a suggestion!

Thanks,
Marc

### PR DESCRIPTION
**Pull request by chr1swallace**

### CODE REVIEW

File: R/check.R [hunk 1]
```diff
@@ -110,6 +111,14 @@ check_dataset <- function(d,suffix="",req=c("snp"),warn.minp=1e-6) {
...
     stop("dataset ",suffix,": ","type must be quant or cc")
 
+  ## varbeta should be > 0
+  if("varbeta" %in% nd & !all(varbeta > 0))
+	  stop("varbeta should be strictly > 0")
+
+  ## MAF should be > 0
+  if("MAF" %in% nd & !all(MAF > 0))
+	  stop("MAF should be strictly > 0")
+
```
Comment (pc494) [inline R/check.R:121]: Does this do something different to the code on lines 85-87?

Comment (chr1swallace) [reply R/check.R:121]: ha! good spot, no it doesn't
do I make a new commit and pull request now, or...?

Comment (pc494) [reply R/check.R:121]: The new commit on the correct branch will auto-update the PR on github once you push it (the PR is "tracking" the branch it comes from)

**Review by pc494 — COMMENTED** [review]
mea culpa.

**Comment by chr1swallace** [comment]
A good reminder that no review is guaranteed to be perfect, but more eyes always better

**Pull request merged**